code/_globalvars/lists/objects.dm

```diff
@@ -80,3 +84,16 @@ GLOBAL_LIST_EMPTY(roundstart_station_borgcharger_areas)
 
 /// List of area names of roundstart station mech rechargers, for the low charge/no charge mech screen alert tooltips.
 GLOBAL_LIST_EMPTY(roundstart_station_mechcharger_areas)
+
+// List of organ typepaths that are not unit test-able, and shouldn't be spawned by some things, such as certain class prototypes.
+GLOBAL_LIST_INIT(prototype_organs, typecacheof(list(
+	/obj/item/organ,
```

Comment:
If I am not mistaken, you are making this typecache contain all organs in existence, because you are not calling it with `only_root_path = TRUE`. Is this intentional?

Reply:
Oh I realize you just moved this from a static to a global, was this always broken? 😄